macOS/Configuration/App/DuckDuckGo.xcconfig

```diff
@@ -43,7 +43,7 @@ SWIFT_ACTIVE_COMPILATION_CONDITIONS[config=CI][arch=*][sdk=*] = NETP_SYSTEM_EXTE
 SWIFT_ACTIVE_COMPILATION_CONDITIONS[config=Debug][arch=*][sdk=*] = NETP_SYSTEM_EXTENSION DEBUG $(FEATURE_FLAGS)
 SWIFT_ACTIVE_COMPILATION_CONDITIONS[config=Review][arch=*][sdk=*] = NETP_SYSTEM_EXTENSION REVIEW $(FEATURE_FLAGS)
 
-// Install Developer ID build to /Applications/DEBUG/
+// Install to /Applications/DEBUG/
```

Comment:
Adjusting since now this will be true for all builds.

Reply:
😢